### PR TITLE
Describe slab memory layout

### PR DESCRIPTION
**Pull request by MichalBlk**
First of all, there is no such a thing as `pool_item_t` (that's a dead comment). Besides, there is an error in the equations.

### CODE REVIEW

File: sys/kern/pool.c [hunk 1]
```diff
@@ -79,19 +79,18 @@ static void add_slab(pool_t *pool, slab_t *slab, size_t slabsize) {
...
    * With:
    *  - usable = slabsize - sizeof(slab_t)
-   *  - itemsize = sizeof(pool_item_t) + size;
    * ... inequation looks as follow:
    * (1) ntotal * itemsize + (ntotal + 7) / 8 <= usable
    * (2) ntotal * 8 * itemsize + ntotal + 7 <= usable * 8
-   * (3) ntotal * (8 * itemsize + 1) <= usable * 8 + 7
-   * (4) ntotal <= (usable * 8 + 7) / (8 * itemisize + 1)
+   * (3) ntotal * (8 * itemsize + 1) <= usable * 8 - 7
+   * (4) ntotal <= (usable * 8 - 7) / (8 * itemisize + 1)
```
Comment (wiklam) [inline sys/kern/pool.c:90]:
```suggestion
   * (4) ntotal <= (usable * 8 - 7) / (8 * itemsize + 1)
```

**Review by wiklam — APPROVED** [review]
Not sure if we need such calculations in code, but your change is ok!

**Comment by cahirwpz** [comment]
Since I've already merged those changes as a part of another PR... could you actually draw a diagram that depicts single slab contents?

**Comment by MichalBlk** [comment]
Fine.

**Comment by cahirwpz** [comment]
We're going to tackle several issues with slab allocator in near future. Hence I close this PR.

**Pull request closed without merging**